openmetadata-ui/src/main/resources/ui/src/components/AddGlossary/AddGlossary.component.tsx

```diff
@@ -153,181 +102,180 @@ const AddGlossary = ({
       layout={PageLayoutType['2ColRTL']}
       pageTitle={t('label.add-entity', { entity: t('label.glossary') })}
       rightPanel={fetchRightPanel()}>
-      <div className="tw-form-container">
+      <div className="tw-form-container glossary-form">
```

Comment:
We can get rid off tw- classes

Reply:
Can't remove it as the entire form layout design is coming from here.